sphinx-docs/Requirements.md

```diff
@@ -44,8 +44,19 @@ Let's look at the **Impersonate User** ability from Stockpile as an example.
 
 Notice in the ability command, two facts `host.user.name` and `host.user.password` will be used. The `paw_provenance`
 requirement enforces that only `host.user.name` facts that were discovered by the agent running the ability can be used
-(i.e. fact originated from the same `paw`). Additionally, the `basic` requirement enforces that only `host.user.name`
-facts with an existing `has_password` relationship to an existing `host.user.password` fact may be used.
+(i.e. fact originated from the same `paw`). In the scenario this ability is run against two agents on two different
+hosts where multiple `host.user.name` and `host.user.password` facts were discovered, the `paw_provenance` prevents
+`host.user.name` and `host.user.password` facts discovered by the second agent on the second host from being used by
+the first agent on the first host. Thus, the agent would only be able to use the `host.user.name` and
+`host.user.password` facts that it discovered itself, emulating the natural operation flow of an adversary (they would
+not be able to use information that they did not discover themselves).
```

Comment:
It's not necessarily about emulating the natural operation flow of an adversary - it's basically making sure we don't use local information from the wrong machine (e.g. local file paths and local user accounts may be different from machine to machine, so they can't be shared globally across all agents in an operation)